src/client/helpers/entity.js

```diff
@@ -225,3 +226,59 @@ export function getISBNOfEdition(entity) {
 export function getEditionFormat(entity) {
 	return (entity.editionFormat && entity.editionFormat.label) || '?';
 }
+
+// relationshipTypeId = 10 refers the relation (<Work> is contained by <Edition>)
+const relationshipTypeId = 10;
+
+/**
+ * Filter the relationship according to relationshipTypeId
+ *
+ * @param {object} entity - Entity with all relationships
+ * @returns {object} retrun the all relationshiops after removing the relatioship with relationshipTypeId = 10
+ */
+export function getFilteredRelationship(entity) {
+	return entity.relationships.filter((relation) => relation.typeId !== relationshipTypeId);
```

Comment:
I'd suggest a defensive approach and checking if entity.relationships is an array, returning and empty array if it is not the case.